app/views/govuk_publishing_components/components/docs/document_list.yml

```diff
@@ -1,16 +1,16 @@
 name: Document list
 description: An ordered list of documents including a document type, when updated and a link.
 body: |
-  Outputs a list to documents, based on an array of document data. This may include:
+  Outputs a list to documents, based on an array of document data. A "document" in this context is considered any page on the web, so the document list can be considered a list of references (links, titles etc) to web pages.
```

Comment:
Is it "any page on the web" or any page on GOV.UK? Can I instead suggest:

`A "document" in this context can be an asset (such as a PDF or other downloadable document) or a web page.`

Reply:
I'm going to amend this to `ODT` instead of `PDF` as a micro effort towards throwing PDFs away since they're inaccessible.